osgp/shared/shared/src/test/java/org/opensmartgridplatform/shared/security/RsaEncryptionServiceTest.java

```diff
@@ -107,7 +126,7 @@ private RsaEncryptionService createRsaEncryptionServiceFromStoredKeys() {
         return randomBytes;
     }
 
-    public static void createNewKeysForTests() throws IOException {
+    public static void createNewKeysForTests() throws IOException, EncrypterException {
```

Comment:
In test code, where an exception is just another test failure, it might be simpler not to do to much bookkeeping about specific exceptions, and just `throw Exception` if any checked exceptions are involved.
That being said, I don't see where `createNewKeysForTests` is used. So it may be better to just remove the method.